src/app/pages/components/components.component.ts

```diff
@@ -8,6 +8,7 @@ import { ButtonComponent } from 'src/app/components/button/button.component';
 import { BreadcrumbsComponent } from 'src/app/components/breadcrumbs/breadcrumbs.component';
 import { CheckboxComponent } from 'src/app/components/checkbox/checkbox.component';
 import { CardComponent } from 'src/app/components/card/card.component';
+import TextareaComponent from 'src/app/components/textarea/textarea.component';
```

Comment:
Is there a reason why this import is not fragmented while the above imports are? :)

Reply:
Yes, the Textarea has its class as a default export. This was a tip that came from Gökberk and Tommy. Could be updated for the other components but not super urgent.

Reply:
I love learning new stuff 😄